### PR TITLE
moved multi requirement in create_rollup_tables script

### PR DESCRIPTION
**Pull request by Nokondi**
RED team was missing some data from the contained_levels table. The script originally only pulled contained levels if they were of type Multi or if they were part of a BubbleChoice level. This script moves the Multi requirement so that all contained levels should now be imported.

## Links
[Slack thread](https://codedotorg.slack.com/archives/C045UAX4WKH/p1715098215080409)

## Testing story

Will confirm with RED team after implementation that they are getting all of the necessary data.

## Follow-up work
Additional changes may be required depending on the needs of RED team for analysis.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by nataliazm99 — APPROVED** [review]
Looks good!

File: bin/cron/create_rollup_tables [hunk 1]
```diff
@@ -68,6 +67,7 @@ def main
           contained_level_text: sanitize_string_for_db(contained_question)
         }
 
+        next unless contained_level.type == 'Multi'
```
Comment (davidsbailey) [inline bin/cron/create_rollup_tables]: `Match` and `EvaluationMulti` also have "answers". do we want to include those too?

here are the counts of the types of the sublevels of level groups:
```
[development] dashboard > LevelGroup.all.map(&:levels).flatten.compact.map(&:type).tally
=> {"Multi"=>7499, "FreeResponse"=>581, "Match"=>52, "TextMatch"=>24, "EvaluationMulti"=>74}
```

Comment (nataliazm99) [reply bin/cron/create_rollup_tables]: Yes please, that would help us validate answers for assessments

Comment (Nokondi) [reply bin/cron/create_rollup_tables]: OK, I added those to the check.

**Review by davidsbailey — APPROVED** [review]
Looks great!

**Pull request merged**